docker/docker-compose.middleware.yaml

```diff
@@ -61,7 +61,7 @@ services:
       # You can generate a strong key using `openssl rand -base64 42`.
       API_KEY: dify-sandbox
       GIN_MODE: 'release'
-      WORKER_TIMEOUT: 15
```

Comment:
I recommend not using such a long timeout for the following reasons:

1. The sandbox is designed for lightweight code execution; heavy tasks should be handled outside of Dify.
2. A long timeout increases risks, such as hackers initiating numerous requests to other servers (DDoS attacks) or running infinite loops within the sandbox, rendering it unusable for other users.

By the way, could you share your specific use case?